src/misc/helper.ts

```diff
@@ -89,22 +118,22 @@ export const saveBinaryHex = (
  * @param {string} filename
  * @returns {number} filesize in bytes
  */
-const getFilesize = (filename: string): number => {
-  const stats = statSync(filename);
-  const fileSizeInBytes = stats.size;
+const getFilesize = async (filename: string): Promise<number> => {
+  const size = await stats(filename);
+  const fileSizeInBytes = size.size;
   return fileSizeInBytes;
 };
 
 /**
  *
  * @author Flo Dörr
  */
-const createDir = () => {
-  if (!existsSync(JITCOIN_DIR)) {
-    mkdirSync(JITCOIN_DIR);
+const createDir = async () => {
+  if (!(await pathExists(JITCOIN_DIR))) {
+    await mkdirp(JITCOIN_DIR);
   }
-  if (!existsSync(BLOCKCHAIN_DIR)) {
-    mkdirSync(BLOCKCHAIN_DIR);
+  if (!await pathExists(BLOCKCHAIN_DIR)) {
```

Comment:
Is there a reason why this call is not wrapped in parentheses? Let's keep it consistent.
```suggestion
  if (!(await pathExists(BLOCKCHAIN_DIR))) {
```